Lib9c/TableData/Sheet.cs

```diff
@@ -102,6 +110,7 @@ public void Set(string csv, bool isReversed = false)
             }
 
             PostSet();
+            _csv = csv;
```

Comment:
`_csv` isn't changed by `Set<T>()`. is it okay?

Reply:
yes. `Set<T>` is only used when apeend inherited sheet(e.g QuestSheet, ItemSheet)